Fixes rubocop.yml on latest rubocop version

naps62 opened the pull request:
The current rubocop.yml throws an error because one of the cops we had enabled
has been split into two.

azevedo-252 commented on deploy/README.md:
```diff
@@ -6,7 +6,7 @@ A guide for deploying web apps in a sane and repeatable way
 # Server setup
 
-* Use [DigitalOcean](https://www.digitalocean.com/)
+* Use [DigitalOcean](https://www.digitalocean.com/). Be sure to select the `Enable Backups` feature, which can only be enabled upon creation
```
Not sure if ```upon``` might create confusion. What about ```on```, or ```during```?

naps62 replied:
changed to during

azevedo-252 commented:
:+1:

naps62 committed:
Fixes rubocop.yml on latest rubocop version

The current rubocop.yml throws an error because one of the cops we had
enabled has been split into two.
